datamade/data-collab-marketing
# Initialize Netlify deployments and update site

## Summary

- Initialize static site deployments with Netlify
- Update logos for new partners and move founding members to a new section

Closes #5 

## Demo

New logos section:

![partners](https://user-images.githubusercontent.com/14170650/54778914-4d22c480-4bec-11e9-85c7-b185a258c986.png)

## Testing instructions

- Visit the deploy preview: https://deploy-preview-6--chicagodatacollaborative.netlify.com/
- Confirm that everything looks good
